mina.netty/src/main/java/org/kaazing/mina/core/session/AbstractIoSession.java

```diff
@@ -426,9 +430,9 @@ public WriteFuture write(Object message, SocketAddress remoteAddress) {
         try {
```

Comment:
It is there to make sure to catch application errors. I guess we can remove the while try { } block.

But if possible we should limit this change to websocket transport only. Can you paste the complete stack trace  ?